frontend/src/__tests__/cypress/cypress/pages/pipelines/topology.ts

```diff
@@ -192,6 +193,30 @@ class PipelineRunJobDetails extends RunDetails {
   selectActionDropdownItem(label: string) {
     this.findActionsDropdown().click().findByRole('menuitem', { name: label }).click();
   }
+
+  mockEnableJob(job: PipelineRunJobKFv2, namespace: string) {
+    return cy.interceptOdh(
+      'POST /api/service/pipelines/:namespace/:serviceName/apis/v2beta1/recurringruns/:recurringRunId',
+      {
+        path: {
+          namespace,
+          serviceName: 'dspa',
+          recurringRunId: `${job.recurring_run_id}:enable`,
+        },
+      },
+      { data: {} },
+    );
+  }
```

Comment:
instead of making the data accept unknown, can you just create a new entry in commands/odh.ts for a new route

`POST /api/service/pipelines/:namespace/:serviceName/apis/v2beta1/recurringruns/:recurringRunId:enable`

Reply:
you will probably need to update the interceptOdh command to accept a `customVerb` param (https://cloud.google.com/apis/design/custom_methods)

so then you would be able to define it properly

Reply:
This was addressed.